dataprep/datasets/_base.py

```diff
@@ -0,0 +1,69 @@
+"""This module implements load dataset related functions"""
+
+import os
+from os.path import dirname
+from typing import List
+
+import pandas as pd
+
+
+def get_dataset_names() -> List[str]:
```

Comment:
Are we supposed to put the data in https://github.com/sfu-db/dataprep-data?

Reply:
No. The data set is stored in the same repo, under `data` folder.